### ISSUE TITLE
Screen readers don't read signpost content when opened

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3523]
## Describe the bug

Screen readers don't read signpost content when opened nor do they know what the trigger buttons will do.

## How to reproduce

https://stackblitz.com/github/clane/clarityTest2/tree/master/clarity-test-app

- Using a screen reader activate the Signpost link under Signpost Demos
- Observe that you hear nothing

## Expected behavior

The signpost content should be read and should indicate the trigger behavior.

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

Do the following for the signpost trigger button.

1) Provide an accessible name that describes what the button does for the user using aria-label. Of course its purpose is for toggling the visibility of hidden content but we probably want something user friendly. 

2) Manage aria-expanded on the signpost-trigger button below.

Do the following for the signpost close button.

1) Ensure that when the signpost close button is activated, aria-expanded is set to false on the signpost-trigger button
2) Focus should return to the signpost-trigger button after the signpost is closed by the signpost close button. 

Finally, use aria-controls on both the signpost trigger button and signpost close button to create a reference between the signpost content and these two controls that toggle its readability.
